scala/scala-impl/src/org/jetbrains/plugins/scala/lang/formatting/settings/ImportsPanel.java

```diff
@@ -273,7 +273,7 @@ private static void setValue(final JRadioButton rb, final boolean value) {
         final JPanel panel3 = new JPanel();
         panel3.setLayout(new GridLayoutManager(1, 1, new Insets(0, 0, 0, 0), -1, -1));
         panel2.add(panel3, new GridConstraints(0, 0, 1, 1, GridConstraints.ANCHOR_CENTER, GridConstraints.FILL_BOTH, GridConstraints.SIZEPOLICY_CAN_SHRINK | GridConstraints.SIZEPOLICY_CAN_GROW, GridConstraints.SIZEPOLICY_CAN_SHRINK | GridConstraints.SIZEPOLICY_CAN_GROW, null, null, null, 0, false));
-        panel3.setBorder(IdeBorderFactory.PlainSmallWithIndent.createTitledBorder(BorderFactory.createEtchedBorder(), this.$$$getMessageFromBundle$$$("messages/ScalaBundle", "imports.panel.classes.to.use.only.with.prefix"), TitledBorder.DEFAULT_JUSTIFICATION, TitledBorder.DEFAULT_POSITION, null, null));
```

Comment:
To avoid such changes you need to enable the internal mode.
See README.md:
https://github.com/JetBrains/intellij-scala/blob/idea243.x/README.md

> After building the project you see git local changes in ImportsPanel.java (or similar files). All IdeBorderFactory.PlainSmallWithIndent are replaced with BorderFactory
Solution: enable [internal mode](https://plugins.jetbrains.com/docs/intellij/enabling-internal.html).
UI Designer uses different border class in internal mode, see com.intellij.uiDesigner.make.FormSourceCodeGenerator#borderFactoryClassName